docs/idiomatic/000keep-minimum-state-variables.md

```diff
@@ -0,0 +1,149 @@
+# Idiom 0: Keep state variables to the minimum
+
+## Description
+
+_A good TLA+ specification minimizes the computation state and makes it visible_.
+
+TLA+ does not have special syntax for assignments to the variables.  For a good
```

Comment:
```suggestion

TLA+ does not have special syntax for variable assignment.  For a good
```

Reply:
BTW, this statement is controversial. TLA+ does have variable assignment, which is what idiom 001 states explicitly:

> TLA+ uses the same glyph, `=`, for assignment, asserting equality, and binding variables.

Perhaps we can rephrase the whole sentence to be consistent with idiom 001.

Reply:
Perhaps something like this?

```suggestion

TLA+ does not have special syntax that distinguishes mutable variable assignment from equality constraints.  For a good
```